### PR TITLE
Enable reverse display of the Max7219 digit

### LINKED ISSUE #14: MAX7219 In Dot Matrix Configuration
**Is your feature request related to a problem/use-case? Please describe.**

add max7219 dot matrix support.

**Describe the solution you'd like:**

home assistan publishing "events,massages.. on max7219 dot matrix .

**Additional context:**

sending text to display on dot matrix.

### PR DESCRIPTION
**Pull request by rspaargaren**
## Description:

**Related issue (if applicable):** fixes https://github.com/esphome/feature-requests/issues/14 See comment of @adadrag

**Pull request in [esphome-docs](https://github.com/esphome/esphome-docs) with documentation (if applicable):** esphome/esphome-docs#726

## Checklist:
  - [X ] The code change is tested and works locally.
  - [X ] Tests have been added to verify that the new code works (under `tests/` folder).

If user exposed functionality or configuration variables are added/changed:
  - [X ] Documentation added/updated in [esphome-docs](https://github.com/esphome/esphome-docs).

### CODE REVIEW

File: esphome/components/max7219digit/display.py [hunk 1]
```diff
@@ -56,6 +58,7 @@ def to_code(config):
     cg.add(var.set_scroll_delay(config[CONF_SCROLL_DELAY]))
     cg.add(var.set_scroll(config[CONF_SCROLL_ENABLE]))
     cg.add(var.set_scroll_mode(config[CONF_SCROLL_MODE]))
+    cg.add(var.set_scroll(config[CONF_REVERSE_ENABLE]))
```
Comment (OttoWinter) [inline esphome/components/max7219digit/display.py:61]:
```suggestion
    cg.add(var.set_reverse(config[CONF_REVERSE_ENABLE]))
```

**Comment by rspaargaren** [comment]
When can this PR be merged and closed?

**Comment by rspaargaren** [comment]
@glmnet Hi do you know when the open PR are going to be integrated into dev? Can u push this enhancement of previous released max display?

**Review by jesserockz — APPROVED** [review]
Thanks

**Pull request merged**